codeSnippets/snippets/tutorial-server-db-integration/src/test/kotlin/com/example/ApplciationTest.kt

```diff
@@ -0,0 +1,108 @@
+package com.example
+
+import com.example.model.Priority
+import com.example.model.Task
+import io.ktor.client.call.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.ktor.server.config.*
+import io.ktor.server.testing.*
+import kotlin.test.*
+
+class ApplicationTest {
+    @Test
+    fun tasksCanBeFoundByPriority() = testApplication {
+        environment {
+            config = MapApplicationConfig()
+        }
```

Comment:
If I'm not mistaken, I believe these environment blocks are not required anymore.

Reply:
Indeed, with these changes they are unnecessary. I updated the steps.